TombEngine/Game/control/box.cpp

```diff
@@ -1432,69 +1437,119 @@ void FindAITarget(CreatureInfo* creature, short objectNumber)
 	}
 }
 
-void FindAITargetObject(CreatureInfo* creature, int objectNumber)
+void FindAITargetObject(ItemInfo& item, GAME_OBJECT_ID objectID)
```

Comment:
Why function argument was changed from `CreatureInfo` to `ItemInfo`?

Reply:
Sezz did it :x